src/workerd/io/BUILD.bazel

```diff
@@ -20,36 +20,34 @@ wd_cc_library(
     # HACK: Currently, the `io` and `api` packages are interdependent. We fold all the sources
     #   from `api` into `io`. In principle, it should be possible to pull them apart so `api`
     #   depends on `io` but not vice versa.
-    # TODO(cleaunp): Fix this.
+    # TODO(cleanup): Fix this.
     srcs = glob(
         ["*.c++"],
         exclude = [
             "worker-interface.c++",
             "*-test.c++",
             "trace.c++",
+            "io-gate.c++",
+            "observer.c++",
+            "actor-cache.c++",
+            "actor-sqlite.c++",
+            "actor-storage.c++",
+            "worker-entrypoint.c++",
         ],
     ) + ["//src/workerd/api:srcs"],
     hdrs = glob(
         ["*.h"],
-        # `trace.h` should really be excluded here, but doing so causes the build to fail on
-        # Windows with errors like:
-        #
-        # ```
-        # ERROR: C:/.../src/workerd/io/BUILD.bazel: Compiling src/workerd/io/compatibility-date.c++ failed: undeclared inclusion(s) in rule '//src/workerd/io:io':
-        # this rule is missing dependency declarations for the following files included by 'src/workerd/io/compatibility-date.c++':
-        #   'bazel-out/x64_windows-opt/bin/src/workerd/io/_virtual_includes/io/workerd/io/trace.h'
-        # In file included from <built-in>:429:
-        # ```
-        #
-        # This feels like a weird interaction between symlinks on Windows and Bazel.
-        # `trace.h` is `#pragma once`d though, so including it in both `io` and `trace` targets
```

Comment:
Isn't this context useful to keep?

Reply:
This context would be helpful, but the underlying issue appears to be fixed – I was able to add trace.h to the exclusions list here and the Windows build is still working. My guess is that this got fixed in Bazel at some point.